### ISSUE TITLE
Mean and Standard Deviation

### ISSUE THREAD

**Issue by anas-zafar** [BrianG13/FLEX#11]
Thank you for the amazing work! I was running the evaluation script. It seems you are using gt bones standard deviation for both GTbones2skel and bones2skel method. Could you please guide me what parameters could I use for inference. I tried using the mean of the predicted bones, no luck so far. @sigal-raab @BrianG13 
`                skeleton = GTbones2skel(fakes_bones_list[i].clone().detach(), norm_parameters[2], norm_parameters[3],
                                        n_joints=self.config.arch.n_joints,
                                        start_position_as_totem=starting_pos_totem) `

**Comment by BrianG13** [comment]
Hi @anas-zafar , sorry for the delay, I was in vacation.
Here is a link to google drive with the norm_parameters list: [link](https://drive.google.com/file/d/1ygEy5gBw4zd8xZjt84aYYmx3fRXbFIfP/view?usp=sharing) 

You can load it on code using the follow:
```
import pickle
with open(path_to_file, "rb") as f:
   norm_parameters = pickle.load(f)
```
